WireDomain/Sources/WireDomain/Repositories/UpdateEventsRepository.swift

```diff
@@ -85,11 +123,23 @@ final class UpdateEventsRepository: UpdateEventsRepositoryProtocol {
             selfClientID: selfClientID,
             sinceEventID: lastEventID
         ) {
+            let batchCount = envelopes.count
+            var count = 0
+            WireLogger.sync.debug("received batch of \(batchCount) envelopes")
+
+            // If we need to abort, do it before processing the next page.
+            try Task.checkCancellation()
+
             for envelope in envelopes {
```

Comment:
wording: we call it envelope but it comes from getUpdateEvents, which could be confusing, not sure what's better to stick with UpdateEvents or rename to getEnvelopes

Reply:
I would prefer to rename the method `getEventEnvelopes`.

Reply:
Though I'm not sure exactly which method you're referring to.